pyttb/sptensor.py

```diff
@@ -688,16 +924,43 @@ def isequal(self, other: Union[sptensor, ttb.tensor]) -> bool:
 
     def logical_and(self, B: Union[float, sptensor, ttb.tensor]) -> sptensor:
         """
-        Logical and with self and another object
+        Logical AND with a scalar or another sparse or dense tensor.
 
         Parameters
         ----------
-        B:
-            Other value to compare with
+        Scalar (float) value or other tensor to compare against.
```

Comment:
By not indenting or providing the parameter name sphinx takes this whole line to be the parameter name.
If we wanted to we could rename `B` to `other` to match `isequal` which might be clearer.
```suggestion
        B:
           Other value to compare against.
```